sdk/ai/azopenai/models.go

```diff
@@ -1578,6 +1598,9 @@ type CompletionsOptions struct {
 	// A collection of textual sequences that will end completions generation.
 	Stop []string
 
+	// Options for streaming response. Only set this when you set stream: true.
```

Comment:
StreamOptions should be moved to GetCompletionsStreamOptions - we provide two separate functions where one of them enables streaming and the other doesn't and this option doesn't apply to the non-streaming mode.